mobile/lib/common/settings_screen.dart

```diff
@@ -165,7 +165,7 @@ class _SettingsScreenState extends State<SettingsScreen> {
               final List<int> bytes = utf8.encode(logsAsString);
               final Directory tempDir = await getTemporaryDirectory();
               String now = DateFormat('yyyy-MM-dd_HH:mm:ss').format(DateTime.now());
```

Comment:
Since we are on it :) 
```suggestion
String now = DateFormat('yyyy-MM-dd_HHmmss').format(DateTime.now());
```

`:` are weirdly expressed as `/` which is super annoying.